lib/src/posthog_flutter_web_handler.dart

```diff
@@ -94,6 +94,9 @@ Future<dynamic> handleWebMethodCall(MethodCall call, JsObject context) async {
         call.arguments['key'],
       ]);
       break;
+    case 'getSessionId':
+      final sessionId = analytics.callMethod('get_session_id');
+      return sessionId;
```

Comment:
https://github.com/PostHog/posthog-js/blob/6da785b18c526f5008c2ec9f0bc98555ca784404/src/posthog-core.ts#L1616
sessionId can be empty, lets return null if its empty instead